src/loyalty-program/index.tex

```diff
@@ -0,0 +1,14 @@
+%textidote: ignore begin
+\section{Loyalty Program}\label{sec:loyalty-program}
+%textidote: ignore end
+
+As mentioned in the problem statement~\ref{subsec:problem-statement} we wanted to look into the effectiveness
```

Comment:
we should not use words such as "we", as it is better to keep it more formal through "the authors" or avoiding making use of pronouns.